### ISSUE TITLE
Snes9x 1.56: BS-X titles no longer boot up/immediate black screen

### ISSUE THREAD

**Issue by roflcopter777** [snes9xgit/snes9x#330]
Snes9x: 1.56 64-bit
ROM: Chrono  Trigger Music Library
Region: NTSC-J
OS: Windows 10 Home 64-bit
Checksum: OK
Error: First it gives an error about the SRAM, then about the BS-X Satellaview and then results in a black screen. Is the BS-X ROM needed to boot Satellaview games? Thanks :3
The issue has been present since Snes9x 1.55

**Comment by bearoso** [comment]
Yes, there is a process needed to boot BS-X games, and it requires the BS-X BIOS and SRAM. You can also use the translated BS-X BIOS. You will need to run the BS-X bios on its own to generate the SRAM before anything else.

See: http://www.snes9x.com/phpbb3/viewtopic.php?p=56616#p56616

**Comment by roflcopter777** [comment]
The game doesn't need S-RAM, at least the Chrono Trigger Music Library. Do the BIOS files need to go in a particular folder? I feel like an idiot for not thinking of this before.

**Comment by bearoso** [comment]
You need the SRAM because the BIOS needs the SRAM for configuration. Read the link I posted for where they go.
